packages/amplify-category-function/src/provider-utils/awscloudformation/utils/constants.ts

```diff
@@ -3,4 +3,12 @@ import { category } from '../../../constants';
 export const categoryName = category;
 export const serviceName = 'Lambda';
 export const functionParametersFileName = 'function-parameters.json';
+export const parametersFileName = 'parameters.json';
 export const provider = 'awscloudformation';
+export const MINUTES = 'Minutes';
+export const HOURLY = 'Hourly';
+export const DAILY = 'Daily';
+export const WEEKLY = 'Weekly';
+export const MONTHLY = 'Monthly';
+export const YEARLY = 'Yearly';
+export const CUSTOM = 'Custom AWS cron expression';
```

Comment:
I think these values would be better as a [string enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)